setup.cfg

```diff
@@ -31,7 +31,6 @@ ignore = W503,W504
 
 [isort]
 include_trailing_comment = True
-known_first_party = btb
```

Comment:
I don't have any objection to remove it, but I when we added it, it was for a good reason.

Reply:
The was a reason and it was a good reason: it failed otherwise.
But the reason of the failure was that the tox.ini configuration was not right, as the project was not being installed in the flake8 environment.
Now the tox.ini config has been fixed, so this can be removed.